src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -808,6 +821,7 @@ internal void RescheduleBufferedExternalEvents()
                     // Need to round-trip serialization since SendEvent always tries to serialize.
                     string rawInput = events.Dequeue();
                     JToken jsonData = JToken.Parse(rawInput);
+                    this.IncrementActionsOrThrowException();
```

Comment:
I probably wouldn't count this one since the user doesn't have direct control over this call. It's something that happens in the background without them really knowing about it. Basically, this code is for handling the case where an orchestration receives an external event but doesn't actually process it. This can happen in certain `ContinueAsNew` scenarios. We try to "reschedule" it for the next iteration of the orchestration.